src/blosc2/lazyexpr.py

```diff
@@ -128,6 +169,30 @@ def save(self, **kwargs):
           if its source is a :ref:`SChunk`, :ref:`NDArray` or a :ref:`C2Array` (see :func:`blosc2.open` notes
           section for more info).
         * This is currently only supported for :ref:`LazyExpr`.
+
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> dtype = np.float64
+        >>> shape = [3, 3]
+        >>> size = shape[0] * shape[1]
+        >>> a = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> b = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> # Define file paths for storing the arrays
+        >>> a_path = 'a_array.b2nd'
+        >>> b_path = 'b_array.b2nd'
+        >>> a1 = blosc2.asarray(a, urlpath=a_path, mode='w')
+        >>> b1 = blosc2.asarray(b, urlpath=b_path, mode='w')
```

Comment:
As `a_path` and `b_path` are going to be used just once, it is better to use the values directly.  In this case, the above *5 lines* can be reduced to:

```
a1 = blosc2.asarray(a, urlpath='a_array.b2nd', mode='w')
b1= blosc2.asarray(b, urlpath='b_array.b2nd', mode='w')
```

and it reads better IMO.